### PR TITLE
fix(reset): remove box-sizing - INNO-948

### PR DESCRIPTION
**Pull request by yhuard**
The goal of this PR is to remove the global `box-sizing` rules from the reset. I also took the opportunity to "upgrade" some of the components I've been fixing

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 832d31d9aa5d2cd0b24a24bb25ab1a64202b0503

https://deploy-preview-627--europa-component-library.netlify.com

**Comment by yhuard** [comment]
I've run the failing tests locally, they pass

File: src/generic/generic-layout-grid/mixins/_grid.scss [hunk 1]
```diff
@@ -74,6 +77,7 @@
 }
 
 @mixin ecl-make-col($size, $columns: $ecl-grid-columns) {
+  // box-sizing: border-box;
```
Comment (degliwe) [inline src/generic/generic-layout-grid/mixins/_grid.scss]: Remove commented code please

File: src/generic/generic-layout-grid/mixins/_grid.scss [hunk 2]
```diff
@@ -95,6 +99,7 @@
 }
 
 @mixin ecl-make-col-modifier($type, $size, $columns) {
+  // box-sizing: border-box;
```
Comment (degliwe) [inline src/generic/generic-layout-grid/mixins/_grid.scss]: Remove commented code please

File: src/generic/generic-layout-grid/mixins/_grid.scss [hunk 3]
```diff
@@ -74,6 +77,7 @@
 }
 
 @mixin ecl-make-col($size, $columns: $ecl-grid-columns) {
+  // box-sizing: border-box;
   flex: 0 0 percentage($size / $columns);
   // width: percentage($size / $columns);
```
Comment (degliwe) [inline src/generic/generic-layout-grid/mixins/_grid.scss]: Remove commented code please

**Review by degliwe — APPROVED** [review]

**Pull request merged**